index.html

```diff
@@ -14,6 +18,128 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+    <header id="header">
+        <nav class="flex">
+            <a href="#header">Intro</a>
+            <a href="#history">History</a>
+            <a href="#skills">Skills</a>
+            <a href="#contact">Contact</a>
+            <a href="https://ericbanker12.github.io/portfolio-website/">Portfolio</a>
+        </nav>
+    </header>
+    <section id="intro" class="flex intro">
+        <img src="images/me.jpg" alt="Eric Banker Photograph">
+        <div>
+            <h1>Hello, I am Eric Banker.</h1>
+            <h3>I am an aspiring web and software developer.</h3>
+            <p>Initially, I majored in biology to pursue a career in research. However, I felt frustrated with the career prospects and slow progression, so I chose to instead pursue a career in web and software development.</p>
+        </div>
+    </section>
+    <section id="history" class="history">
```

Comment:
Good use of ids! Ids are useful not just for styling, but also for navigation '' '